src/win/tty.c

```diff
@@ -1850,7 +1883,14 @@ static int uv_tty_write_bufs(uv_tty_t* handle,
                continue;
             }
 
-          } else if (utf8_codepoint == ';') {
+          /* SP ; is invalid */
+          } else if (utf8_codepoint == ';' && prev_utf8_codepoint != ' ') {
+            /* Check for this is after space */
+            if (prev_utf8_codepoint == ' ') {
```

Comment:
This check is tautological - you already know `prev_utf8_codepoint` is not a space per the if statement above.